chicago/bills.py

```diff
@@ -97,11 +96,15 @@ def scrape(self):
                                                            'communication', 
                                                            'report', 
                                                            'oath of office') :
-                    continue
+                    bill_type = None
                 else :
                     bill_type = legislation_summary['Type'].lower()
 
-                bill_session = self.session(legislation_summary['Intro\xa0Date'])
+                try :
+                    bill_session = self.session(self.toTime(legislation_summary['Intro\xa0Date']))
+                except :
+                    print(legislation_summary)
```

Comment:
Oh, maybe also a description of the problem. So raise AssertionError("Time parsing failed. Legislation Summary: {}".format(legislation_summary))